bleach/__init__.py

```diff
@@ -43,6 +44,8 @@
 
 ALLOWED_STYLES = []
 
+ALLOWED_PROTOCOLS = copy.copy(HTMLSanitizer.acceptable_protocols)
```

Comment:
If you decide to change the default protocols to a smaller list I would suggest a big BACKWARDS INCOMPATIBLE warning on the next release, as suddenly all sorts of links would stop working in people's code unless they also went through the protocols one by one and checked if anyone used them or not. We have people using at least five of the ones removed.